persimmon/website/models.py

```diff
@@ -11,6 +11,19 @@ class User(models.Model):
     name = models.CharField(max_length=200)
     username = models.CharField(max_length=200)
     employee_level = models.IntegerField(choices=EmployeeLevel.choices)
+     
+    #returns true if employee is set at given security level
+    def check_level(self, level):
+        if(self.employee_level==level):
+            return True
+        else:
+            return False
+
+    @property
+    def get_name(self):
+        return
+    def get_level(self):
```

Comment:
You don't need to write getters for attributes - this is python, not Java; it's okay to have direct member access.